mix.lock

```diff
@@ -10,8 +10,9 @@
   "makeup_elixir": {:hex, :makeup_elixir, "0.14.1", "4f0e96847c63c17841d42c08107405a005a2680eb9c7ccadfd757bd31dabccfb", [:mix], [{:makeup, "~> 1.0", [hex: :makeup, repo: "hexpm", optional: false]}], "hexpm", "f2438b1a80eaec9ede832b5c41cd4f373b38fd7aa33e3b22d9db79e640cbde11"},
   "mime": {:hex, :mime, "1.3.1", "30ce04ab3175b6ad0bdce0035cba77bba68b813d523d1aac73d9781b4d193cf8", [:mix], [], "hexpm", "6cbe761d6a0ca5a31a0931bf4c63204bceb64538e664a8ecf784a9a6f3b875f1"},
   "nimble_parsec": {:hex, :nimble_parsec, "0.6.0", "32111b3bf39137144abd7ba1cce0914533b2d16ef35e8abc5ec8be6122944263", [:mix], [], "hexpm", "27eac315a94909d4dc68bc07a4a83e06c8379237c5ea528a9acff4ca1c873c52"},
-  "opentelemetry": {:hex, :opentelemetry, "0.4.1", "519f0e2daa89d1c4a0fb37310db9828ee89181e6cff349f6b1fdbda2fc4441e5", [:rebar3], [{:opentelemetry_api, "~> 0.4.0", [hex: :opentelemetry_api, repo: "hexpm", optional: false]}], "hexpm", "afba124ebe368fd530db30347ec3bf41f489a3d9b245bfd8397d5753b83e3aae"},
-  "opentelemetry_api": {:hex, :opentelemetry_api, "0.4.1", "afce47063e099407daa105bf25e2aae75822f3c25b61a213f9328ff40ae6c448", [:mix, :rebar3], [], "hexpm", "7b057ca9bd5f2220d02fe2337630979df27c8d0fefc6eba371fb3cb202eb618e"},
+  "opentelemetry": {:hex, :opentelemetry, "0.5.0", "6a7072a6f54b8be07a42dd2aaca1ee0389038681405d0a27c4c5e5325b6c1a82", [:rebar3], [{:opentelemetry_api, "~>0.5.0", [hex: :opentelemetry_api, repo: "hexpm", optional: false]}], "hexpm", "a45261c421240199184d55447108f1d9f49c6b2b0e0e2056aaaa1a1fe77d2eda"},
+  "opentelemetry_api": {:hex, :opentelemetry_api, "0.5.0", "29ecaa7bb86df4dbd44b14fd71fd4faadff5c08d178545b66b3861bd350be28a", [:mix, :rebar3], [], "hexpm", "fa3c23198e84d6d1cdb8a4408374c8d73c9d2ee7a9e587a6d539def15851ffad"},
+  "opentelemetry_exporter": {:git, "https://github.com/open-telemetry/opentelemetry-erlang.git", "482e8e6f821736771c8099d544492343cd1f3302", [sparse: "apps/opentelemetry_exporter"]},
```

Comment:
How did this get into the lock?

Reply:
Hm, must have accidentally added it to the mix file initially, and then not cleaned up the lock. Removed